cli/src/commands/rebase.rs

```diff
@@ -213,8 +217,14 @@ Please use `jj rebase -d 'all:x|y'` instead of `jj rebase --allow-large-revsets
         ));
     }
 
+    if args.skip_empty {
+        return Err(user_error(
```

Comment:
nit: maybe `cli_error()` is more appropriate (but other deprecation messages use `user_error()` or `ui.warning()`.)

Reply:
I think `cli_error` is probably the right error to return, even if other places have done it differently.